source/quickguides/quick-guide-generic.rst

```diff
@@ -0,0 +1,248 @@
+
+        import os
+        from acme_diags.parameter.core_parameter import CoreParameter
+        from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+        from acme_diags.run import runner
+        
+        param = CoreParameter()
+        
+        param.reference_data_path = '#expand obs_path#time-series/'
+        param.test_data_path = '#expand test_data_path#time-series/E3SM_v1/'
+        param.test_name = 'e3sm_v1'
+        
+        prefix = '#expand web_address#doc_examples/'
+        param.results_dir = os.path.join(prefix, 'area_mean_with_obs')
+        #param.multiprocessing = True
+        #param.num_workers =  40
+        
+        # We're passing in this new object as well, in
+        # addition to the CoreParameter object.
+        
+        ts_param = AreaMeanTimeSeriesParameter()
+        #ts_param.ref_names = ['none']   # This setting plot model data only
+        ts_param.start_yr = '2002'
+        ts_param.end_yr = '2008'
+        
+        runner.sets_to_run = ['area_mean_time_series']
+        runner.run_diags([param, ts_param])
+
+
+This set can also be ran with the core diagnostics sets,
+so that all the plots are shown in one viewer.
+The following is an example to run all sets:
+
+    .. code:: python
+
+        import os
+        from acme_diags.parameter.core_parameter import CoreParameter
+        from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+        from acme_diags.run import runner
+        
+        param = CoreParameter()
+        
+        param.reference_data_path = '#expand obs_path#climatology/'
+        param.test_data_path = '#expand test_data_path#climatology/'
+        param.test_name = '20161118.beta0.FC5COSP.ne30_ne30.edison'
+        param.multiprocessing = True
+        param.num_workers = 40
+        prefix = '#expand html_path#doc_examples'
+        param.results_dir = os.path.join(prefix, 'all_sets')
+        
+        #
+        ##Set specific parameters for new sets
+        ts_param = AreaMeanTimeSeriesParameter()
+        ts_param.reference_data_path = '#expand obs_path#time-series/'
+        ts_param.test_data_path = '#expand obs_path#time-series/E3SM_v1/'
+        ts_param.test_name = 'e3sm_v1'
+        ts_param.start_yr = '2002'
+        ts_param.end_yr = '2008'
+        
+        runner.sets_to_run = ['lat_lon','zonal_mean_xy', 'zonal_mean_2d', 'polar', 'cosp_histogram', 'meridional_mean_2d', 'area_mean_time_series']
+        runner.run_diags([param, ts_param])
+
+
+Advanced: Running custom diagnostics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The following steps are for 'advanced' users, who want to run custom diagnostics.
+So, most users will not run the software like this.
+
+
+By default, with ``e3sm_diags``,
+a built in set of variables are defined for each diagonostics sets.
+To do a short run, e.g. only running through a subset of variables,
```

Comment:
Is running through a subset of variables the only way a run would be short? If so, we should use `i.e.` instead of `e.g.`

Reply:
Same as above.

Reply:
It sounds like there are other ways for a run to be short, so `e.g.` makes sense.